fleet_notifications/state_checker.py

```
@@ -132,12 +132,12 @@ def _check_orders_and_call_if_done(self, new_states: dict[int, OrderState]) -> N
             )
             order = all_orders.get(order_id, None)
             if not order:
-                logger.error(f"Order not found: {order_id}")
+                logger.warning(f"Order not found: {order_id}")
                 continue
             try:
                 car = self.car_api.get_car(order.car_id)
             except:
```

Ruff (0.8.2)

Do not use bare `except`

(E722)